index.html

```diff
@@ -325,5 +325,15 @@ <h6 class="languages-world">Languages</h6>
         </form>
       </div>
     </footer>
+
+    <div class="menu-bar">
+         <i class="fa fa-times"></i>
+        <ul class="header-slide-links">
+          <li><a href="#">Portifolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
Kindly add the blur effect to match the Figma design.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/161656551-f247fb04-ed78-44bc-8a30-ea6aeb4ae4c5.png) | ![image](https://user-images.githubusercontent.com/24703171/161656668-1ead4247-8d99-4e6e-88bc-ac8c886f6f64.png) |